src/diff.rs

```diff
@@ -61,11 +61,75 @@ pub struct ModifiedFile {
 
 #[derive(Debug, PartialEq, Eq)]
 pub struct FileDiff {
-    // TODO
+    pub hunks: Vec<Hunk>,
+}
+
+#[derive(Debug, PartialEq, Eq)]
+pub struct Hunk {
```

Comment:
I'd be interested in thinking of a less `git`ty name. But not opposed to keeping it as `Hunk` either. My suggestion would something along the lines of `FileChange`

Reply:
I'm down, but not sure about introducing "Change" which is kind of another word for "Diff" :thinking: - ie. it'll sound like `FileDiff` which we already have and `FileChange` are the same thing.

Reply:
Maybe something like `DiffSet`?

Reply:
Yeah or perhaps `LineDiffSet`